src/js/facility-locator/components/ServicesAtFacility.jsx

```diff
@@ -131,15 +129,16 @@ class ServicesAtFacility extends Component {
     // TODO: clean up once we have real data
     return (
       <div>
-        {facility.type === 'va_health_facility' ? this.renderHealthServices() : null}
+        <p style={{ margin: '0 0 0.5em' }}>Services current as of <strong>{moment().format('MMMM D, YYYY')}</strong></p>
```

Comment:
won't `moment.().format('MMM D, YYYY')` give us the current date? Do we need to pull this date from the api for each location? Otherwise we probably don't need it on each detail page.